test/rubygems/test_gem_commands_build_command.rb

```diff
@@ -376,7 +376,7 @@ def test_build_signed_gem_with_cert_expiration_length_days
     skip 'openssl is missing' unless defined?(OpenSSL::SSL) && !java_platform?
 
     gem_path = File.join Gem.user_home, ".gem"
-    Dir.mkdir gem_path
+    FileUtils.mkdir_p gem_path
```

Comment:
Shouldn't `Gem.user_home` always exist? Why is this (and the changes analagous to this one) needed?

Reply:
Right, Thanks!